mozilla/fx-private-relay
# Add VPN Promo Banner

Notes: 
- Because Relay is English-only there is no language gate on this promo currently. 

## Testing 

Please test this twice – once logged in and once logged out of FxA / Relay. 

- Load page (logged out)
- **Expected: Banner should be visible** 
- Click "X" link in top right of banner.
- **Expected: Banner should no longer be visible and page should appear as normal (no extra padding)**
- Reload the page
- **Expected: Banner is not visible as you dismissed the banner**
